src/SFA.DAS.AssessorService.Application/Interfaces/IBatchLogRepository.cs

```diff
@@ -10,6 +10,8 @@ namespace SFA.DAS.AssessorService.Application.Interfaces
     public interface IBatchLogRepository
     {
         Task<BatchLog> Create(BatchLog batchLog);
-        Task<ValidationResponse> UpdateBatchLogBatchWithDataRequest(Guid requestId, BatchData requestBatchData);
+        Task<ValidationResponse> UpdateBatchLogSentToPrinter(int batchNumber, DateTime batchCreated, int numberOfCertificates, 
```

Comment:
could we pass a BatchData object of some kind here rather than loads of separate arguments?

Reply:
Done